File: Models/Record.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CrudVueNET.Models
{
    public class Record
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public double Value { get; set; }
        public DateTime? Date { get; set; }
        public string Time { get; set; }
    }
}

File: crud-clientapp/src/models/Record.ts

export class Record {
    public id?: number;
    public name?: string;
    public value: number;
    public date?: string;
    public time?: string;
    constructor(
        value: number,
        name: string,
        date: string,
        time: string,
        id?: number) {
        this.value = value;
        this.name = name;
        this.date = date;
        this.time = time;
        this.id = id;
    }
}


